### DISCUSSION TITLE
Suggestion buttons

### DISCUSSION THREAD

**Discussion by ansa-aboudou** [FlowiseAI/Flowise#1601] [Q&A] [answered]
Hello guys!

I just wanted to ask, do you know how to set up suggestions buttons in Flowise like this :

![image](https://github.com/FlowiseAI/Flowise/assets/54751685/bf4b8f83-52f1-4030-9a1f-7804fca6a464)

Thank you for your help

**Comment by Jaredude** [comment, marked as answer]
Those are Start Prompts. You can configure them by clicking the Settings (gear) button on the flow. 
![Screenshot 2024-01-25 at 2 43 33 PM](https://github.com/FlowiseAI/Flowise/assets/10228950/493428d1-b5a0-455f-9ec5-284cc6891aab)

**Reply by ansa-aboudou** [reply]
Thank you very much !
